sdk/eventhub/Azure.Messaging.EventHubs/samples/Sample04_PublishingEvents.md

```diff
@@ -91,7 +150,60 @@ When publishing events, it may be desirable to request that the Event Hubs servi
                 
 There is no means of predicting which partition will be associated with a given partition key; we can only be assured that it will be a consistent choice of partition.  If you have a need to understand which exact partition an event is published to, you will need to specify the partition directly rather than using a partition key.
 
-When the batch is published, the `EventHubProducerClient` will receive an acknowledgment from the Event Hubs service; so long as no exception is thrown by this call, your application can consider publishing successful.  The service assumes responsibility for delivery of the batch.  All of your event data will be published to one of the Event Hub partitions, thought here may be a slight delay until it is available to be read.
+### Event Hub Buffered Producer Client
+
+When using the `EventHubBufferedProducerClient`, events are enqueued with a partition key option.  Because the producer manages publishing, there is no explicit call.  When the producer is closed, it will ensure that any remaining enqueued events have been published.  All of your event data will be published to one of the Event Hub partitions, thought here may be a slight delay until it is available to be read.
```

Comment:
```suggestion
When using the `EventHubBufferedProducerClient`, events are enqueued with a partition key option.  Because the producer manages publishing, there is no explicit call.  When the producer is closed, it will ensure that any remaining enqueued events have been published.  All of your event data will be published to one of the Event Hub partitions, though there may be a slight delay until it is available to be read.
```